great-idea-website/index.html

```diff
@@ -15,49 +15,92 @@
 </head>
 
   <body>
-
```

Comment:
If you use a "container" or "wrapper" div just inside your <body> if helps to contain everything nicely